src/components/listSelection/Bubble.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Bubble = props => {
+const Bubble = ({letter}) => {
```

Comment:
can this not just be
`Bubble = letter -> {`

Reply:
I tried it and it doesn't work